extensions/integration-tests/src/notebook.util.ts

```diff
@@ -73,14 +74,19 @@ export const pythonKernelMetadata = {
 	}
 };
 
-export function writeNotebookToFile(pythonNotebook: azdata.nb.INotebookContents): vscode.Uri {
+export function writeNotebookToFile(pythonNotebook: azdata.nb.INotebookContents, testName: string): vscode.Uri {
+	let fileName = getFileName(testName);
+	assert(false, 'Didn\'t have file name for notebook');
```

Comment:
This is always going to fail? Don't you want filename !== undefined && filename !== '' or something similar?